MEMBERSHIP.md

```diff
@@ -1,7 +1,20 @@
 # Membership in the SFOSC
 
 Welcome! Anyone who is interested in contributing to the SFOSC in any way is welcome
-to become a member. At the moment, the only privilege membership brings is being able
+to become a member.
+
+Membership is **not a requirement** for participating in the community.
+Members may be consulted about governance of the community. For example through votes.
+The [CONTRIBUTING GUIDELINES](https://github.com/sfosc/sfosc/blob/master/CONTRIBUTING.md)
+explains several ways you can contribute regardless of membership.
```

Comment:
"Guidelines explain" not "guidelines explains"

Reply:
Good point, fixed this.